slick/src/main/scala/slick/util/InternalArrayQueue.scala

```diff
@@ -0,0 +1,175 @@
+package slick.util
+
+import java.util
+
+private[util] final class InternalArrayQueue[E >: Null <: AnyRef](capacity: Int) {
```

Comment:
Oh, it looks like it comes from `ManagedArrayBlockingQueue` (and I have deja-vu feeling like I asked the same question about a previous version...)